actions/websocket_actions.jsx

```diff
@@ -184,23 +184,27 @@ export function reconnect(includeWebSocket = true) {
         timestamp: Date.now(),
     });
 
-    loadPluginsIfNecessary();
+    const state = getState();
+    const currentTeamId = state.entities.teams.currentTeamId;
+
+    loadPluginsIfNecessary().then(() => {
+        if (currentTeamId) {
```

Comment:
Since this is run asynchronously, we should call `getState` in here instead of outside. `currentTeamId` may have changed between the time the websocket reconnects to when the plugins are reloaded